django_ses/__init__.py

```diff
@@ -45,6 +45,13 @@ def dkim_sign(message, dkim_domain=None, dkim_key=None, dkim_selector=None, dkim
     return message
 
 
+def cast_nonzero_to_float(val):
+    """Cast nonzero number to float; on zero or None, return None"""
+    if not val:
+        return None
```

Comment:
Could check `if val is None` to allow 0.

Alternatively, could `return val` inside the `if`.

Reply:
@pcraciunoiu the way you use `self._throttle` relies on truthiness too, though, ie:

`if self._throttle:`

So if the value at that point is 0 or None, the outcome seems like it'd be the same; throttling is skipped. So it would seem fine if this PR is merged, since it will just commute 0 to None. Thoughts?

Reply:
Good point. It's also used here https://github.com/django-ses/django-ses/blob/77c170adbfc2695256cfa31a6d7ad8569f049c84/django_ses/__init__.py#L178
But that's only if it's truthy.

So it should be fine... We can give this a go.

Reply:
Also FWIW I didn't write this library, just took it over when it wasn't maintained.